WordPress/WordPressTest/ViewRelated/Post/Controllers/PostListViewControllerTests.swift

```diff
@@ -62,10 +62,15 @@ class PostListViewControllerTests: CoreDataTestCase {
 
         // When: Simulate a post being created
         // Then: This should not cause a crash
-        expect {
+        //
+        // Note that `XCTAssertNoThrow` catches `NSException`s as well as Swift's `throw`.
+        //
+        // This test originally used Nimble's `raiseException` but that matcher is no longer available in the SPM build.
+        // See https://github.com/Quick/Nimble/blob/e313d9a67ec2e4171d416c61282e49fc3aadc7a4/Sources/Nimble/Matchers/RaisesException.swift#L1
+        XCTAssertNoThrow(try {
             _ = PostBuilder(self.mainContext, blog: blog).with(status: .draft).build()
-            try! self.mainContext.save()
-        }.notTo(raiseException())
+            try self.mainContext.save()
+        }())
```

Comment:
Took me a while to figure this one out. From the commit message:

You can verify the change works by modifying the source code with the following diff:

```diff
--- a/WordPress/WordPressTest/PostBuilder.swift
+++ b/WordPress/WordPressTest/PostBuilder.swift
@@ -200,6 +201,7 @@ class PostBuilder {
     func build() -> Post {
         // TODO: Enable this assertion once we can ensure that the post's MOC isn't being deallocated after the `PostBuilder` is
         // assert(post.managedObjectContext != nil)
+        NSException(name: .genericException, reason: "testing").raise()
         return post
     }
```

You can also verify the `XCTAssertNoThrow` behavior in isolation with a
test like the following:

```swift
func test_XCTAssertNoThrow_failsWhenExceptionRaised() throws {
    XCTExpectFailure("Expected failure to verify XCTAssertNoThrow fails when an exception raises")
    XCTAssertNoThrow(NSException.fixture().raise())
}
```

Reply:
Looks okay to me. 🙂